pandas/tests/series/test_timeseries.py

```diff
@@ -370,6 +370,12 @@ def test_pct_change(self, datetime_series):
             rs, (filled / filled.shift(freq="5D") - 1).reindex_like(filled)
         )
 
+    def test_pct_change_with_duplicate_axis(self):
+        # GH 28664
+        Series(range(70), date_range("2019", periods=70, freq="D")).pct_change(
```

Comment:
OK thanks. So can you now:

  1. Reduce the number of periods from 70 to 3
  2.Assign the result of this to a variable called `result`
  3. On the next line say `expected = ...` replacing ... with the actual values this should yield
  4. After that do `tm.assert_series_equal(result, expected)`